phangsPipeline/scTheoreticalNoise.py

```diff
@@ -0,0 +1,201 @@
+# Routines to build a theoretical noise map
+
+import logging
+
+import numpy as np
+
+import scipy.ndimage as nd
+import scipy.ndimage.morphology as morph
+import scipy.stats as ss
+from scipy.signal import savgol_coeffs
+
+import astropy.wcs as wcs
+import astropy.units as u
+from astropy.convolution import convolve, Gaussian2DKernel
+from astropy.io import fits
+from astropy.stats import mad_std
+from astropy.table import Table
+
+from matplotlib import pyplot as plt
+
+from spectral_cube import SpectralCube
```

Comment:
For the filename and this import, you're not actually using spectral-cube here. Suggest `utilsTheoreticalNoise` instead for consistency.